dash_table/package-info.json

```diff
@@ -37,50 +37,49 @@
   },
   "author": "Chris P <chris@plot.ly",
   "license": "MIT",
-  "dependencies": {
-    "@types/react": "^16.4.8",
-    "ramda": "^0.25.0",
-    "react": "16.3.0",
-    "react-dom": "16.3.0",
-    "react-select": "^1.2.1",
-    "sheetclip": "^0.3.0"
-  },
+  "dependencies": {},
```

Comment:
There are multiple ways of looking at the meaning of deps and devDeps in the npm world -- but in the end this project only exposes a bundled file containing *all* that's needed and that can be seen as there being only dev dependencies as there is nothing that is required outside that bundle in the production environment / to use the package. This is the approach taken here.